lib/auth/routes_authtype.js

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Do we still need search guard license header for files we create?

Reply:
I saw most modules within this directory had this header. I can remove this if it is no longer required.